modules/EvseV2G/connection/tls_connection.cpp

```diff
@@ -132,17 +154,18 @@ bool build_config(tls::Server::config_t& config, struct v2g_context* ctx) {
             // workaround (see above libevse-security comment)
             const auto key_password = info.password.value_or("");
 
-            config.certificate_chain_file = cert_path.c_str();
-            config.private_key_file = key_path.c_str();
-            config.private_key_password = key_password.c_str();
+            config.chains.emplace_back();
+            config.chains[0].certificate_chain_file = cert_path.c_str();
+            config.chains[0].private_key_file = key_path.c_str();
+            config.chains[0].private_key_password = key_password.c_str();
```

Comment:
You can use an reference `auto&` instead of `config.chains[0]`.
For example:
```
auto& config_chain_0 = config.chains.emplace_back();
config_chain_0.certificate_chain_file = cert_path.c_str();
[...]
```

Reply:
done